great_expectations/checkpoint/actions.py

```diff
@@ -237,7 +237,7 @@ def __init__(
         self.notify_with = notify_with
         self.show_failed_expectations = show_failed_expectations
 
-    def _run(
+    def _run(  # type: ignore[override] # signature does not match parent
```

Comment:
😢 how can we remedy this issue? It feels like a violation of sound OOO design and I'm quite confident there are probably some latent bugs around as a result of it.

Reply:
The difference is `data_asset` no being required and the addition of the optional `payload`.